Mathlib/CategoryTheory/Monoidal/Cartesian/Basic.lean

```diff
@@ -27,7 +27,7 @@ We however develop the theory for any `F.OplaxMonoidal`/`F.Monoidal`/`F.Braided`
 requiring it to be the `ofChosenFiniteProducts` one. This is to avoid diamonds: Consider
 e.g. `𝟭 C` and `F ⋙ G`.
 
-In applications requiring a finite preserving functor to be oplax-monoidal/monoidal/braided,
+In applications requiring a finite-preserving functor to be oplax-monoidal/monoidal/braided,
```

Comment:
This strikes me as a weird term, but perhaps a finite and preserving functor is weirder?

Reply:
Thanks for catching this! I recall this wording gave me pause as well, but I wasn't familiar enough with the subject matter to know for sure it was a mistake.

This is fixed now.